binding/web/README.md

```diff
@@ -0,0 +1,178 @@
+# leopard-web
+
+**NOTE**: This is a beta build.
+
+The Picovoice Leopard library for web browsers, powered by WebAssembly.
+
+This library processes always-listening voice commands in-browser, offline. All processing is done via WebAssembly and Workers in a separate thread.
+
+Looking for Leopard on NodeJS? See the [@picovoice/leopard-node](https://www.npmjs.com/package/@picovoice/leopard-node) package.
+
+## Compatibility
+
+- Chrome / Edge
+- Firefox
+- Safari
+
+This library requires several modern browser features: WebAssembly, Web Workers, IndexedDB and promises. Internet Explorer will _not_ work.
+
+## Installation & Usage
+
+### Package
+
+Install the [Leopard-Web package](https://www.npmjs.com/package/@picovoice/leopard-web) using `yarn`:
+
+```console
+yarn add @picovoice/leopard-web
+```
+
+or using `npm`:
+
+```console
+npm install --save @picovoice/leopard-web
+```
+
+### AccessKey
+
+Leopard requires a valid Picovoice `AccessKey` at initialization. `AccessKey` acts as your credentials when using Leopard SDKs.
+You can get your `AccessKey` for free. Make sure to keep your `AccessKey` secret.
+Signup or Login to [Picovoice Console](https://console.picovoice.ai/) to get your `AccessKey`.
+
+### Leopard Models
+
+Leopard requires a model file on initialization. Get a custom model file from [Picovoice Console](https://console.picovoice.ai/cat)
+or you can use the [default model file](/lib/common/leopard_params.pv).
+For the web packages, there are two methods to initialize Leopard.
+
+#### Public Directory
+
+**NOTE**: Due to modern browser limitations of using a file URL, this method does __not__ work if used without hosting a server.
+
+This method fetches the model file from the public directory and feeds it to Leopard. Copy the model file into the public directory:
+
+```console
+cp ${LEOPARD_MODEL_FILE} ${PATH_TO_PUBLIC_DIRECTORY}
+```
+
+#### Base64
+
+**NOTE**: This method works without hosting a server, but increases the size of the model file roughly by 33%.
+
+This method uses a base64 string of the model file and feeds it to Leopard. Use the built-in script `pvbase64` to
+base64 your model file:
+
+```console
+npx pvbase64 -i ${LEOPARD_MODEL_FILE} -o ${OUTPUT_DIRECTORY}/${MODEL_NAME}.js
+```
+
+The output will be a js file which you can import into any file of your project. For detailed information about `pvbase64`,
+run:
+
+```console
+npx pvbase64 -h
```

Comment:
this package has not released yet, right?

Reply:
it is -> comes with web utils